packages/jest/package.json

```diff
@@ -37,6 +37,7 @@
     "@jest/test-result": "^27.5.1",
     "@jest/types": "^27.5.1",
     "@replay-cli/pkg-build": "workspace:^",
+    "@replay-cli/shared": "workspace:^",
```

Comment:
We get a confusing error if a transitive bundled dependency isn't declared here. While we need this situation to error for regular external dependencies, I think we could compute potential bundled dependencies recursively here:
https://github.com/replayio/replay-cli/blob/229636edcd16a9a3dc11465b447763eff59c34cf/scripts/pkg-build/src/bin.ts#L29-L33

I imagine that this recursive search should include **dependencies** (unlike the top-level one above that looks at `devDependencies`)